license_comment

```diff
@@ -0,0 +1,2 @@
+Copyright 2024 Yahoo Inc.
```

Comment:
was this mistakenly checked in?

Reply:
It's the license file for https://github.com/google/addlicense. I'll add a comment in the readme.